packages/govern-create/test/base.ts

```diff
@@ -0,0 +1,63 @@
+import { deployments, ethers, network, waffle } from 'hardhat'
+import { expect } from 'chai'
+import { GovernBaseFactory, GovernRegistry } from '../typechain'
+
+const EVENTS = {
+  REGISTERED: 'Registered',
+  SET_METADATA: 'SetMetadata',
+}
+
+describe('Govern Base Factory', function () {
+  beforeEach(async () => {
+    await deployments.fixture()
+  })
+
+  const deployDAO = async (
+    useProxies: boolean,
+    gasTarget: number,
+    deployToken = false
+  ) => {
+    const registryDeployment = await deployments.get('GovernRegistry')
```

Comment:
What was the idea behind moving the base test setup to this function instead of keeping it in the beforeEach call?

Reply:
You are right, we should keep it in the `beforeEach`